pkg/pdatatest/ptracetest/testdata/ignore-end-timestamp/expected.yaml

```diff
@@ -16,5 +16,7 @@ resourceSpans:
             name: span1
             parentSpanId: ""
             spanId: fd0da883bb27cd6b
+            endTimeUnixNano: "11651379494838206464"
             status: {}
             traceId: 8c8b1765a7b0acf0b66aa4623fcb7bd5
+
```

Comment:
```suggestion
```
Nit: I see a few empty lines at the end of files here, are they necessary? If they're necessary feel free to ignore all of my corrections 👍